media-source-respec.html

```diff
@@ -2731,11 +2729,13 @@
         </thead>
         <tbody>
           <tr>
-            <td>April 28 2016</td>
+            <td>May 04 2016</td>
             <td>
               <ul>
-                <li>Issue 54 - Clarified use of Presentation Time Stamps (PTS) for <a def-id="buffered"></a>.</li>
-              </ul>
+                <li>Issue 19, 20 &amp; 26 - Resolved problems with Duration Changes and Range Removals.</li>
+                <li>Separated Range Removals from Duration Changes, required separate remove().</li>
+                <li>Disabled abort on Range Removals.</li>
+             </ul>
             </td>
           </tr>
           <tr>
```

Comment:
It looks like Issue 54 was incorrectly removed from the Revision History.  I intend to restore it and add the rawgit link.